src/ui/public/route_based_notifier/__tests__/index.js

```diff
@@ -10,12 +10,15 @@ describe('ui/route_based_notifier', function () {
 
```

Comment:
But what does that have to do with aggregation field options? I'm trying to understand the connection to the purpose of the PR so that I can review it better. If it's not connected, that makes the PR a lot harder to review.